clients/java/src/main/java/io/camunda/zeebe/client/CamundaClientBuilder.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client;
```

Comment:
🔧 The package should also move away from Zeebe and over to Camunda, I guess.

Reply:
This is true for all new `Camunda...` classes, I'd say.

Reply:
You are right, but maybe not for everything. Because it will be an impact for users

Reply:
But for the new classes we should move the package names, shouldn't we?